src/tests/ftest/nvme/nvme_pool_exclude.py

```diff
@@ -0,0 +1,240 @@
+            flags = self.ior_w_flags
+        else:
+            flags = self.ior_r_flags
+
+        # Add a thread for these IOR arguments
+        process = threading.Thread(target=self.ior_thread,
+                                   kwargs={"pool": self.pool,
+                                           "oclass": oclass,
+                                           "api": api,
+                                           "test": test,
+                                           "flags": flags,
+                                           "results":
+                                           self.out_queue})
+        # Launch the IOR thread
+        process.start()
+        # Wait for the thread to finish
+        process.join()
+
+    def run_nvme_pool_exclude(self, num_pool):
+        """Run Pool Exclude
+            Args:
+             int : total pools to create for testing purposes.
+        """
+        # Create a pool
+        pool = {}
+        target_list = []
+
+        # Exclude target : random two targets (target idx : 0-7)
+        n = random.randint(0, 6)
+        target_list.append(n)
+        target_list.append(n+1)
+        t_string = "{},{}".format(target_list[0], target_list[1])
+
+        # Exclude rank :  ranks other than rank 0.
+        exclude_servers = len(self.hostlist_servers) * 2
+        rank_list = list(range(1, exclude_servers))
+
+        for val in range(0, num_pool):
+            pool[val] = TestPool(self.context, dmg_command=self.dmg_command)
+            pool[val].get_params(self)
+            # Split total SCM and NVME size for creating multiple pools.
+            pool[val].scm_size.value = int(pool[val].scm_size.value /
+                                           num_pool)
+            pool[val].nvme_size.value = int(pool[val].nvme_size.value /
+                                            num_pool)
+            pool[val].create()
+
+        for val in range(0, num_pool):
+            self.pool = pool[val]
+            for oclass, api, test in product(self.ior_dfs_oclass,
+                                             self.ior_apis,
+                                             self.ior_test_sequence):
+                self.run_ior_thread("Write", oclass, api, test)
+
+                self.pool.display_pool_daos_space("Pool space: Before Exclude")
+                pver_begin = self.get_pool_version()
+
+                index = random.randint(1, len(rank_list))
+                rank = rank_list.pop(index)
+                self.log.info("Removing rank %d", rank)
+
+                time.sleep(5)
+                self.log.info("Pool Version at the beginning %s", pver_begin)
+                output = self.dmg_command.pool_exclude(self.pool.uuid,
+                                                       rank, t_string)
+                self.log.info(output)
+                fail_count = 0
+                while fail_count <= 20:
+                    rebuild_status = self.get_rebuild_status()
```

Comment:
(pylint-no-member) Instance of 'NvmePoolExclude' has no 'get_rebuild_status' member